content/docs/dvclive/api-reference/set_step.md

```diff
@@ -0,0 +1,54 @@
+# dvclive.set_step()
+
+Signals that the current step has ended and updates `step` with the given value.
+
+```py
+def set_step(step: int):
+```
+
+#### Usage:
+
+```py
+import dvclive
+
+for step in [0, 10, 20]:
+    dvclive.set_step(step)
+    dvclive.log("metric_1", 0.9)
+    dvclive.log("metric_2", 0.7)
+```
+
+## Description
+
+DVCLive uses the `step` value to track the progress of each metric logged with
+`dvclive.log()`. You can use `dvclive.set_step()` to set `step` to any value.
```

Comment:
> set `step` to any value.

Can there be invalid values? Like going backwards, repeating numbers, negatives, floats, etc.

Reply:
We don't currently have any restrictions implemented so technically it can be any of those.

Reply:
Not floats though, right? `step: int`